.CI/create_feedstocks

```diff
@@ -7,14 +7,21 @@ set -e
 # recipes from. Currently this is `master`.
 git checkout "${TRAVIS_BRANCH}"
 
-# Download and setup Miniconda
-mkdir -p ~/miniconda_staging
-pushd ~/miniconda_staging
-curl -L -O https://raw.githubusercontent.com/conda-forge/conda-smithy/e976c7e84bb3c4846e7562afd1a42c4b535b51f5/bootstrap-obvious-ci-and-miniconda.py
-python bootstrap-obvious-ci-and-miniconda.py ~/miniconda x64 3 --without-obvci && source ~/miniconda/bin/activate root
```

Comment:
Had to put a hack in to workaround a feedstock conversion issue. So this conflicts, but it is easy to fix.

xref: https://github.com/conda-forge/staged-recipes/pull/2261